pom.xml

```diff
@@ -236,12 +236,12 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-surefire-plugin</artifactId>
-                    <version>${maven-surefire-plugin.version}</version>
```

Comment:
> Since the conflict didn't show anything related to the new variable for surefire I think I made the correct choice ...

The previous change extracted `<maven-surefire-plugin.version>2.22.1</maven-surefire-plugin.version>` for better reuse. IMO you should better revert this change for `pom.xml` b/c the dependency is already up-to-date.

Reply:
@JCgH4164838Gh792C124B5 any help needed for the rebase/amend?

Reply:
Hello @sepe81 .
I'll try to rebase/amend the change to pom.xml, but it's not something I've done before with Git.  If I don't post an update in the next day or two it's a pretty safe bet I'm stumped.  :)